cli/options.go

```diff
@@ -283,3 +295,15 @@ func getAsStringList(em map[string]string) []string {
 	}
 	return m
 }
+
+// resolvePath resolves "~" in paths
+func resolvePath(path string) (string, error) {
+	if strings.HasPrefix(path, "~") {
```

Comment:
Shouldn't this be `"~/"` since we are doing `path[2]` later

Reply:
should be "~"+filepath.Separator then

Reply:
Yah, and `path[2]` should be... not sure what... I'm not fluent in windows paths but does `~\\test` mean something?

Reply:
should this work on Windows at all? This is not AFAIK a platform convention. Although WSL2 complicates things.

Reply:
It would seem `cd ~` works on PowerShell

Reply:
Actually, I wonder we have to manage this _at all_. Isn't this a shell extension thing, that should take place BEFORE our code get ran ?

Reply:
It's a shell extension thing yes

Reply:
answering myself: we need this to support use of `~/...` inside yaml as referenced files